vlib/v/checker/tests/assign_type_mismatch_with_generics_err.out

```diff
@@ -1,7 +1,7 @@
 vlib/v/checker/tests/assign_type_mismatch_with_generics_err.vv:13:9: error: cannot assign to `b`: expected `bool`, not `fn (Bar) bool`
    11 |     mut b := false
    12 |     if f.f != none {
-   13 |         b = f.f or { panic(err) }
+   13 |         b = f.f
       |               ^
    14 |     } else {
-   15 |         b = true
+   15 |         assert false
```

Comment:
Why did this need to be changed - it does not seem relevant to the current PR?

Reply:
Imho restore this .vv/.out pair.

Reply:
`b` is bool, `f.f` is `?Fn[T]`.  So makes no sense `b = f.f` there.

Reply:
The goal is to check the check error.

Reply:
Well... restoring it.